### DISCUSSION TITLE
Lottie Performance in scrolling

### DISCUSSION THREAD

**Discussion by programmatrix88** [airbnb/lottie-ios#2129] [Q&A]
Hi, 
My task wants me to render different lotties in tableview. The problem is that I am unable to give the configuration to my catalog team on what sizes/types of lottie they should give to me so that when I scroll, I dont get any jerks/frame drops. 
Is there any way to know that?
For what I have learnt:
1. Lotties frames have to rendered under 16ms so that frame drop doesn't happen[am I right?]. But how to measure that ?
2. Does lotties sizes also matters? 
3. Whats up with different lotties compression types like dotlottie etc. does that make any difference to perfromance 
4. Does lotties no of lotties also maters in particular cell? I mean If I render 4lotties in UITAbleviewcell so how to calculate its performance

I am little confused and would like opinion of experts.

**Comment by calda** [comment]
There's some discussion about performance here: https://github.com/airbnb/lottie-ios/discussions/1886#discussion-4702528
